client/my-sites/checkout/composite-checkout/components/wp-line-item.tsx

```diff
@@ -222,18 +196,22 @@ function WPNonProductLineItem( {
 			<span aria-labelledby={ itemSpanId } className="checkout-line-item__price">
 				<LineItemPrice actualAmount={ actualAmountDisplay } isSummary={ isSummary } />
 			</span>
-			{ hasDeleteButton && removeProductFromCart && formStatus === FormStatus.READY && (
+			{ hasDeleteButton && removeProductFromCart && (
 				<>
 					<DeleteButton
-						coupon={ coupon }
 						className="checkout-line-item__remove-product"
-						buttonType="borderless"
+						buttonType={ 'text-button' }
+						aria-label={ String(
+							translate( 'Remove %s from cart', {
+								args: label,
+							} )
+						) }
 						disabled={ isDisabled }
 						onClick={ () => {
 							setIsModalVisible( true );
 						} }
 					>
-						<DeleteIcon uniqueID={ deleteButtonId } product={ label } />
+						{ translate( 'Remove from cart' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **32** times:
`translate( 'Remove from Cart' )` *ES Score: 9*

Reply:
These are the same text with different capitalization, but I think the new version is better because it doesn't unnecessarily capitalize "Cart".